### ISSUE TITLE
Reimplementing Live Points

### ISSUE THREAD

**Issue by casudo** [casudo/Kickbase-Insights#25]
SoonTM the Live Points will be re-added.

**Comment by simonsagstetter** [comment]
@casudo FYI:
I came across that live points are only available with the player event endpoint `competitions/{competitionId}/playercenter/{playerId}`

All other points related endpoints like ranking or teamcenter will be updated after the a match is finished. So if you want to have a representation of live points during a match you got to use this endpoint for each player and sum it up :)

**Comment by casudo** [comment]
@simonsagstetter Thanks, I also came across this endpoint. I bought the Pro abonnement for a month just to get it. ;-)

Can you confirm that this endpoint is also accessable without any active abonnement? Couldn't try this yet.

**Comment by simonsagstetter** [comment]
@casudo Yes you can use this endpoint without an subscription. I have none and I still can access it. In fact it is used when you go to competitions > matchdays > choose a team with a finished match > click on a player -> you can see the player events. They are also coming from that endpoint :)
